src/components/graphqlFormGenerator/FormInput.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <!-- NOTE: the is field comes from `props` -->
+  <!-- eslint-disable-next-line vue/require-component-is -->
+  <component
+   v-model="model"
+   v-bind="props"
+   :label="label"
+   :gqlType="gqlType"
+  />
+</template>
+
+<script>
+import Vue from 'vue'
+
+import { VTextField } from 'vuetify/lib/components/VTextField'
+import GList from '@/components/graphqlFormGenerator/components/List'
```

Comment:
Probably for later. It's working now, but this brings a risk of running into some errors with webpack due to circular dependencies. This module (`FormInput`) has a dependency on the default export of `List`. And `List` has a dependency on `FormInput`.

There are other circular dependencies in this package, which could indicate that we need to move things into a separate module.

https://railsware.com/blog/how-to-analyze-circular-dependencies-in-es6/

(BTW, I might have introduced some circular dependencies in other components too, so that's something we can review later and fix when/if possible I think :grimacing: )

Reply:
Hmmm, tricky.

FormInput and List are recursive components so one requires the ability to create the other.

Not sure how to get rid of that circular dependency.

Reply:
It works for now, so we don't need to worry for 0.2. We can push to the 1.0 backlog.

Reply:
I've stuck up an issue tagged against `1.0`.